roles/container-engine/containerd/templates/config.toml.j2

```diff
@@ -0,0 +1,21 @@
+# Kubernetes doesn't use containerd restart manager.
+disabled_plugins = ["restart"]
+
+[debug]
+  level = "info"
+
+[plugins.linux]
+  shim = "/usr/bin/containerd-shim"
+  runtime = "/usr/sbin/runc"
+
+[plugins.cri]
```

Comment:
It would be nice to have here also:
sandbox_image = "{{ pod_infra_image_repo }}:{{ pod_infra_image_tag }}"

otherwise it seems useless to override pod_infra_image_repo at all - containerd will use default pause image

Reply:
Thanks, I added them. I also merge some of configuration from your repository.
Initial I merge your PR but I lost them while I rebase and refactor, sorry.